dependencies.props

```diff
@@ -1,14 +1,71 @@
-       <JitVersion>2.0.0-preview2-25226-02</JitVersion>
+    <CoreFxCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreFxCurrentRef>
+    <CoreClrCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreClrCurrentRef>
+    <StandardCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</StandardCurrentRef>
   </PropertyGroup>
-
-  <!-- Create a collection of all project.json files for dependency updates. -->
+  
+  <!-- Auto-upgraded properties for each build info dependency. -->
+  <PropertyGroup>
+	<CoreFxExpectedPrerelease>preview2-25301-02</CoreFxExpectedPrerelease>
+	<CoreClrExpectedPrerelease>preview2-25301-02</CoreClrExpectedPrerelease>
+    <StandardExpectedPrerelease>preview2-25301-01</StandardExpectedPrerelease>
+  </PropertyGroup>
+  
+  <!-- Package dependency verification/auto-upgrade configuration. -->
+   <PropertyGroup>
+      <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
+      <DependencyBranch>master</DependencyBranch>
+      <CurrentRefXmlPath>$(MSBuildThisFileFullPath)</CurrentRefXmlPath>
+  </PropertyGroup>
+  
   <ItemGroup>
-    <_ProjectJsonExcludes Include="$(SourceDir)test\**\project.json" />
-    <_ProjectJsonExcludes Include="$(SourceDir)pkg\**\project.json" />
-    <ProjectJsonFiles Include="$(SourceDir)**\project.json" Exclude="@(_ProjectJsonExcludes)" />
-  </ItemGroup>
+	<RemoteDependencyBuildInfo Include="CoreFx">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)corefx/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreFxCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+	<RemoteDependencyBuildInfo Include="CoreClr">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)coreclr/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+	<RemoteDependencyBuildInfo Include="Standard">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)standard/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+
+	<DependencyBuildInfo Include="@(RemoteDependencyBuildInfo)">
+		<RawVersionsBaseUrl>https://raw.githubusercontent.com/dotnet/versions</RawVersionsBaseUrl>
+	</DependencyBuildInfo>
+	
+	<XmlUpdateStep Include="CoreFx">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>CoreFxExpectedPrerelease</ElementName>
+		<BuildInfoName>CoreFx</BuildInfoName>
+	</XmlUpdateStep>
+	<XmlUpdateStep Include="CoreClr">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>CoreClrExpectedPrerelease</ElementName>
+		<BuildInfoName>CoreClr</BuildInfoName>
+	</XmlUpdateStep>
+    <XmlUpdateStep Include="Standard">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>StandardExpectedPrerelease</ElementName>
+		<BuildInfoName>Standard</BuildInfoName>
+	</XmlUpdateStep>
+</ItemGroup>
+  
+  <!-- Copied Microsoft.NETCore.Runtime.CoreCLR and Microsoft.NETCore.Runtime.JIT versions -->
+  <PropertyGroup>
+       <CoreCLRVersion>2.0.0-$(CoreClrExpectedPrerelease)</CoreCLRVersion>
```

Comment:
You should use an XmlUpdateStep to create these properties instead of relying on the fact they are always 2.0.0? See https://github.com/dotnet/corefx/blob/master/dependencies.props#L99 for example.